### PR TITLE
Mark paid resources

### PR DESCRIPTION
**Pull request by ffaf1**
As requested in #155.

### CODE REVIEW

File: site/documentation.markdown [hunk 1]
```diff
@@ -6,27 +6,29 @@ isDocumentation: true
 
 # Documentation
 
+Resources marked with \[$$\] are paid.
```
Comment (tomjaguarpaw) [inline site/documentation.markdown:9]: Perhaps "require payment" rather than "are paid".

File: site/documentation.markdown [hunk 2]
```diff
@@ -52,7 +54,7 @@ Short, dense, classic ways to hit the ground running
 ## Online Resources
 
 Curated resources put together by Haskellers:
-
+
```
Comment (tomjaguarpaw) [inline site/documentation.markdown:57]: Would you mind undoing this whitespace change? Not a big deal but personally I tend to prefer diffs that are as clear as possible.

**Comment by tomjaguarpaw** [comment]
Cool, thank you! 

I've made a couple of minor comments but no need to address them until some others have had the chance to give their thoughts too.

File: site/documentation.markdown [hunk 3]
```diff
@@ -6,27 +6,29 @@ isDocumentation: true
...
 
 *   [Developing Web Applications with Haskell and Yesod](http://www.yesodweb.com/book)
-*   [Functional Design and Architecture](https://www.manning.com/books/functional-design-and-architecture)
-*   [Haskell in Depth](https://www.manning.com/books/haskell-in-depth)
+*   \[$$\] [Functional Design and Architecture](https://www.manning.com/books/functional-design-and-architecture)
+*   \[$$\] [Haskell in Depth](https://www.manning.com/books/haskell-in-depth)
 *   [Parallel and Concurrent Programming in Haskell](https://simonmar.github.io/pages/pcph.html)
-*   [Practical Haskell](https://www.apress.com/gp/book/9781484244791)
-*   [Production Haskell](https://leanpub.com/production-haskell)
+*   \{$$\] [Practical Haskell](https://www.apress.com/gp/book/9781484244791)
```
Comment (tomjaguarpaw) [inline site/documentation.markdown:30]: `{` should be `[`

File: site/documentation.markdown [hunk 4]
```diff
@@ -6,27 +6,29 @@ isDocumentation: true
...
+*   \[$$\] [Get Programming with Haskell](https://www.manning.com/books/get-programming-with-haskell)
 
 ## Intermediate Haskell Books
 
 *   [Developing Web Applications with Haskell and Yesod](http://www.yesodweb.com/book)
-*   [Functional Design and Architecture](https://www.manning.com/books/functional-design-and-architecture)
-*   [Haskell in Depth](https://www.manning.com/books/haskell-in-depth)
+*   \[$$\] [Functional Design and Architecture](https://www.manning.com/books/functional-design-and-architecture)
+*   \[$$\] [Haskell in Depth](https://www.manning.com/books/haskell-in-depth)
 *   [Parallel and Concurrent Programming in Haskell](https://simonmar.github.io/pages/pcph.html)
```
Comment (tomjaguarpaw) [inline site/documentation.markdown:29]: Is it really possible to read Parallel and Concurrent Programming in Haskell without payment? The link has a further "Read the book online" link to https://www.oreilly.com/library/view/parallel-and-concurrent/9781449335939/, but that requires an O'Reilly "free trial". I guess technically it doesn't require payment, but I don't know how much or for how long you can read without payment.

I *thought* the book was previously freely available online without paying anything, but if it was I can't find out how now ...

Comment (tomjaguarpaw) [reply site/documentation.markdown:29]: Oh, it seems the contents of the book *is* available on that O'Reilly page, but nonetheless it keeps asking you to sign up for a free trial :)

Comment (ffaf1) [reply site/documentation.markdown:29]: If you click on “Table of contents” and then on any section, you can read it seems you can read it at will, without logging in. Whether it is a bug or a feature I don’t know!

**Comment by tomjaguarpaw** [comment]
FYI we're waiting for other members of the committee to give their reviews.

**Review by tomjaguarpaw — APPROVED** [review]

**Review by romefeller — APPROVED** [review]
Nice idea. I will point out to my book also.

**Pull request merged**